packages/cosmwasm-stargate/src/signingcosmwasmclient.ts

```diff
@@ -233,8 +229,14 @@ export class SigningCosmWasmClient extends CosmWasmClient {
   ) {
     super(tmClient);
     const {
-      registry = createDefaultRegistry(),
-      aminoTypes = new AminoTypes({ ...createWasmAminoConverters(), ...createBankAminoConverters() }),
+      registry = new Registry([
+        ...defaultStargateTypes,
+        ...wasmTypes
```

Comment:
```suggestion
        ...wasmTypes,
```

this should make prettier happy